README.md

```diff
@@ -42,6 +42,12 @@ Receives `getState` function for  accessing current store state and `action` obj
 
 *Default: `null` (always log)*
 
+#### __transform (Function)__
+Transform state before print. Eg. convert Immutable object to plain JSON.
+
+*Default: `null` (no transform)*
```

Comment:
Easier to have `identity` function as default, and then there's no need for ternary expressions in the source?